### PR TITLE
Delete note - done.

### PR DESCRIPTION
**Pull request by varanthir**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -89,10 +90,24 @@ function cancelNewNote() {
     UI.form.style.visibility = 'hidden';
```
Comment (ulfryk) [inline script.js]: do not use inline styles - use css classes

File: script.js [hunk 2]
```diff
@@ -89,10 +90,24 @@ function cancelNewNote() {
...
+    saveData(MYAPP.notes);
+    showNotes(UI.notes, MYAPP.notes);
+}
+
 // UI
+function addDelEvents(notesList) {
+    notesList.forEach((note, index) =>
+        document.querySelector(`.del-btn-${index}`)
+        .addEventListener('click', () => { delNote(note); }));
+}
```
Comment (ulfryk) [inline script.js]: this should be indented like:

```
notesList.forEach((note, index) =>
    document.querySelector(`.del-btn-${index}`)
        .addEventListener('click', () => { delNote(note); }));
```

or for better readability:

```
notesList.forEach((note, index) => document
    .querySelector(`.del-btn-${index}`)
    .addEventListener('click', () => { delNote(note); }));
```

**Comment by ulfryk** [comment]
Looks Good 👍

**Comment by varanthir** [comment]
Thanks

**Pull request merged**